gateway/1.8/modules/ROOT/pages/flex-gateway-istio.adoc

```diff
@@ -0,0 +1,540 @@
+= Integrating Flex Gateway with Istio Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Integrate Flex Gateway into your Istio Service Mesh to leverage Flex Gateway policies, logging, and metrics from the Anypoint Platform control plane. Deploy Flex Gateway as an ingress gateway to manage incoming north-south cluster traffic and internal east-west traffic inside the cluster.
+
+To deploy this Flex Gateway use case, use the following architecture diagram and tutorial. The use case is a modified version of the https://istio.io/latest/docs/examples/bookinfo/[Istio Bookinfo Application] with Flex Gateway integrated. In the use case:
```

Comment:
Instead of calling this the "use case", can you just say something like "integrate Flex Gateway with ...."?